examples/minioTelemetryService/mockdevice.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
```

Comment:
let's use the default `"net/http"` to achieve this to avoid third party package import that is only used for this example